app/src/components/widgets/tables/productStatus/innerComponents.jsx

```diff
@@ -0,0 +1,290 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { formatItemName } from 'controllers/testItem';
+import { LAUNCHES_PAGE, TEST_ITEM_PAGE } from 'controllers/pages';
+import { AbsRelTime } from 'components/main/absRelTime';
+import { NameLink } from 'pages/inside/common/nameLink';
+import { ExecutionStatistics } from 'pages/inside/common/launchSuiteGrid/executionStatistics';
+import { DefectStatistics } from 'pages/inside/common/launchSuiteGrid/defectStatistics';
+import { StatisticsLink } from 'pages/inside/common/statisticsLink';
+import { DefectLink } from 'pages/inside/common/defectLink';
+import { formatStatus } from 'common/utils/localizationUtils';
+import { FILTER_TITLE_TYPE, FILTER_TYPE, TOTAL_TYPE } from './constants';
+import {
+  defaultStatusesMessages,
+  defaultStatisticsMessages,
+  passingRateMessage,
+  hintMessages,
+  totalMessage,
+  filterMessage,
+} from './messages';
+import { getStatisticsStatuses } from '../launchesTable/utils';
+
+import styles from './innerComponents.scss';
+
+const cx = classNames.bind(styles);
+
+export const NameColumn = (
+  { className, value },
+  name,
+  { formatMessage, linkPayload, onFilterNameClick },
+) => {
+  if (value.type === TOTAL_TYPE) {
+    return (
+      <div className={cx('name-col', 'total-col', 'total-col--desktop', className)}>
+        {formatMessage(totalMessage)}
+      </div>
+    );
+  }
+
+  if (value.type === FILTER_TITLE_TYPE) {
+    return (
+      <div className={cx('filter-name-col', className)}>
+        <span className={cx('prefix-text')}>{formatMessage(filterMessage)}:&nbsp;</span>
+        <span className={cx('filter-name-text')}>{value.name}</span>
+      </div>
+    );
+  }
+
+  const ownLinkParams = {
+    page: value.type === FILTER_TYPE ? LAUNCHES_PAGE : TEST_ITEM_PAGE,
+    payload: { ...linkPayload },
+  };
+
+  return (
+    <div
+      className={cx('name-col', className)}
+      onClick={() => {
+        if (value.type === FILTER_TYPE) onFilterNameClick(value.filterId);
+      }}
+    >
+      <div className={cx('main-info')}>
+        <NameLink itemId={value.linkId} ownLinkParams={ownLinkParams} className={cx('name-link')}>
+          <span title={value.name} className={cx('name')}>
+            {`${value.name ? formatItemName(value.name) : ''} `}
+          </span>
+        </NameLink>
+        <span className={cx('number-box')}>
+          <NameLink itemId={value.linkId} ownLinkParams={ownLinkParams} className={cx('name-link')}>
+            {value.number && <span className={cx('number')}>#{value.number}</span>}
```

Comment:
Which purpose here to render the `value.number` not near the [`value.name`](https://github.com/reportportal/service-ui/pull/1762/files#diff-30677d043f8af9b7050f24e0d006b5e6R66) in one `NameLink` component?